[2022-08-06 19:38:57,606] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 19:38:57,624] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 19:38:57,624] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:57,624] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:57,625] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:57,650] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-16 00:00:00+00:00
[2022-08-06 19:38:57,660] {standard_task_runner.py:52} INFO - Started process 2838 to run task
[2022-08-06 19:38:57,662] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '874', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpki2e5n9p', '--error-file', '/tmp/tmp6bz54zsq']
[2022-08-06 19:38:57,663] {standard_task_runner.py:80} INFO - Job 874: Subtask second_task
[2022-08-06 19:38:57,734] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:57,826] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-08-06 19:38:57,827] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:57,828] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:57,841] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:57,843] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:57,843] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:57,874] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220516T000000, start_date=20220806T193857, end_date=20220806T193857
[2022-08-06 19:38:57,915] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:57,945] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:51,301] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 21:52:51,317] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 21:52:51,317] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:51,317] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:51,317] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:51,334] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-16 00:00:00+00:00
[2022-08-06 21:52:51,341] {standard_task_runner.py:52} INFO - Started process 3048 to run task
[2022-08-06 21:52:51,343] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '880', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpk3bav8ef', '--error-file', '/tmp/tmp7ckxl38t']
[2022-08-06 21:52:51,344] {standard_task_runner.py:80} INFO - Job 880: Subtask second_task
[2022-08-06 21:52:51,436] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:51,559] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-08-06 21:52:51,560] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:51,561] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:51,580] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:51,582] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:51,582] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:51,619] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220516T000000, start_date=20220806T215251, end_date=20220806T215251
[2022-08-06 21:52:51,676] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:51,729] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:15,721] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 21:58:15,746] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 21:58:15,746] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:15,746] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:15,746] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:15,772] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-16 00:00:00+00:00
[2022-08-06 21:58:15,779] {standard_task_runner.py:52} INFO - Started process 2834 to run task
[2022-08-06 21:58:15,784] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '878', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4fafh5p5', '--error-file', '/tmp/tmpzauw4isj']
[2022-08-06 21:58:15,785] {standard_task_runner.py:80} INFO - Job 878: Subtask second_task
[2022-08-06 21:58:15,864] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:15,968] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-08-06 21:58:15,970] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:15,971] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:15,990] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:15,993] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:15,993] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:16,034] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220516T000000, start_date=20220806T215815, end_date=20220806T215816
[2022-08-06 21:58:16,077] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:16,121] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:15,481] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 22:13:15,497] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 22:13:15,498] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:15,498] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:15,498] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:15,518] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-16 00:00:00+00:00
[2022-08-06 22:13:15,525] {standard_task_runner.py:52} INFO - Started process 2837 to run task
[2022-08-06 22:13:15,528] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '876', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3my32rhu', '--error-file', '/tmp/tmpr615nfje']
[2022-08-06 22:13:15,528] {standard_task_runner.py:80} INFO - Job 876: Subtask second_task
[2022-08-06 22:13:15,591] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:15,680] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-08-06 22:13:15,680] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:15,681] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:15,693] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:15,695] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:15,695] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:15,724] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220516T000000, start_date=20220806T221315, end_date=20220806T221315
[2022-08-06 22:13:15,779] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:15,818] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:34,769] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 22:24:34,783] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-08-06 22:24:34,783] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:34,783] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:34,783] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:34,798] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-16 00:00:00+00:00
[2022-08-06 22:24:34,804] {standard_task_runner.py:52} INFO - Started process 2890 to run task
[2022-08-06 22:24:34,806] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '881', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp8r1_pn05', '--error-file', '/tmp/tmp12v_d5ch']
[2022-08-06 22:24:34,806] {standard_task_runner.py:80} INFO - Job 881: Subtask second_task
[2022-08-06 22:24:34,894] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-16T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:35,051] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-08-06 22:24:35,053] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:35,054] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:35,072] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:35,074] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:35,074] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:35,123] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220516T000000, start_date=20220806T222434, end_date=20220806T222435
[2022-08-06 22:24:35,181] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:35,235] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
